Header header
# sensor_msgs/JointState joint_states

float32[4] pel_pos_cmd
float32[4] pel_vel_cmd
float32[3] foot_left_cmd
float32[3] foot_right_cmd
float64[127] mpc_solution
float32[2] f_init
float32[3] mpc_pel_pos
float32[4] obs_info 
